index.html

```diff
@@ -0,0 +1,135 @@
+                <p class="name">Hi! I'm <span>Arinze I.C</span></p>
+                <p class="desc">Front-end web developer</p>
+                <a href="#">My Work</a>
+            </div>
+        </section>
+    </div>
+
+        <section id="social-media">
+            <h3>Let's connect:</h3>
+            <div>
+                <a href="https://www.instagram.com/ari.n.ze/" target="_blank"><i class="fab fa-instagram"></i></a>
+                <a href="https://www.linkedin.com/in/arinze-ihemedu-14b738148/" target="_blank"><i class="fab fa-linkedin-in"></i></a>
+                <a href="https://www.quora.com/profile/Arinze-Ihemedu" target="_blank"><i class="fab fa-quora"></i></a>
+                <a href="https://github.com/Arinzemike1" target="_blank"><i class="fab fa-github"></i></a>
+            </div>	
+        </section>
+
+        <section id="about">
+            <div class="container">
+                <h1>About Me</h1>
+                <div class="row">
+                    <div class="col-md-6 text-center about-col">
+                        <div class="profile">
+                            <img src="picpic.jpg" class="user">
+                            <p>Hi there! I'm Arinze Ihemedu, a graduate of applied informatics and system programming from Vitebsk State University. I've got some basic understanding of c++ programming language but here I am doing what I love most which is web development. My stack: HTML5, CSS3/Bootstrap, Javascript/Jquery.</p>
+                        </div>
+                    </div>
+
+                    <div class="col-md-6 skills-bar">
+                        <p>HTML</p>
+                        <div class="progress">
+                            <div class="progress-bar" style="width: 85%;">80%</div>
+                        </div>
+                        <p>CSS</p>
+                        <div class="progress">
+                            <div class="progress-bar" style="width: 70%;">60%</div>
+                        </div>
+                        <p>Javascript</p>
+                        <div class="progress">
+                            <div class="progress-bar" style="width: 40%;">40%</div>
+                        </div>
+                        <p>Wordpress</p>
+                        <div class="progress">
+                            <div class="progress-bar" style="width: 70%;">70%</div>
+                        </div>
+                        <p>GIT</p>
+                        <div class="progress">
+                            <div class="progress-bar" style="width: 30%;">50%</div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div class="contact-btn"><a href="#">Contact me</a></div>
+        </section>
+    </main>
+	
+<footer class=" footer page-footer font-small blue pt-4">
+
+	<div class="container-fluid text-center text-md-left">
+	  <div class="row">
+		<div class="col-md-6 mt-md-0 mt-3 lg">
```

Comment:
I think i have been seing this for a while now. On this line of code, is this the proper indentation as compared to that on the preceeding line (line 98)? Do re-visit to verify and effect all amendments where necessary.